Samples/WinMLSamplesGallery/WinMLSamplesGallery/SampleMetadata/SampleMetadata.json

```diff
@@ -71,6 +71,17 @@
       "CSharpGithubLink": "https://github.com/microsoft/Windows-Machine-Learning/blob/master/Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/EncryptedModel/EncryptedModel.xaml.cs",
       "Docs": [],
       "IsRecentlyAdded": true
+    },
+    {
+      "Title": "Adapter Selection",
+      "DescriptionShort": "Select different adapters based on your power and performance needs.",
+      "Description": "The sample showcases how to use Windows ML with different adapters that have tradeoffs between power and performance.",
```

Comment:
I didn't know adapters = a gpu or cpu device, could the description have a bit of an explanation on adapters?

Reply:
I'm honestly still trying to understand what exactly constitutes an adapter myself. The answers that I've seen online seem a little vague so I'd be open to suggestions from anyone as to what a good description might be.

Reply:
could we use "device" instead?